addons/sale_loyalty/models/sale_order.py

```diff
@@ -1350,7 +1351,7 @@ def _try_apply_program(self, program, coupon=None):
                     "This discount (%(discount)s) is not compatible with \"%(other_discount)s\". "
                     "Please remove it in order to apply this one.",
                     discount=global_reward.description,
-                    other_discount=applied_global_reward.program_id.reward_ids.description,
+                    other_discount=applied_global_reward.program_id.reward_ids.mapped('description')
```

Comment:
```suggestion
                    other_discount=applied_global_reward.description
```
We want description of applied reward